src/card.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  flex-wrap: wrap;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  margin: 40px 16px 0;
+  line-height: 18px;
+  color: var(--color-dark);
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--color-gray);
+  margin: 4px 16px 0;

```

Comment:
You also could add padding to the card and didn't add a horizontal margin to every child component - it's optional